portfolio.html

```diff
@@ -0,0 +1,231 @@
+          <img src="./images/Union.png" class="log" />
+        </div>
+        <ul class="links">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li>
+            <a class="fa-regular fa-envelope"></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section class="headline1">
+      <article>
+        <h1>I’m Gakii<br />Glad to see,You!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work
+          and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
+        <a href="#">LET’S CONNECT</a>
+      </article>
+        <div class="social-media">
+          <ul>
+            <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/Linkedin%20icon.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/medium.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/angellist.svg" alt=""></a></li>
+          </ul>
+        </div>
+    </section>
+    <section class="grid_container">
+      <div class="block1">
+        <div class="block1a">
+          <img src="images/Snapshoot Portfolio1.png" />
+        </div>
+        <div class="item1b">
+          <h2>Tonic</h2>
+          <div class="frame2">
+            <span>CANOPY</span>
+            <ul class="role">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button type="button"><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="item2">
+        <div class="item2a">
+          <img src="images/Snapshoot%20Portfolio2.png">
+        </div>
+        <div class="item1b">
+          <h2>Multi-Post Stories</h2>
+          <div class="frame2">
+           <span>FACEBOOK</span>
+            <ul class="role">
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button type="button"><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] on desktop version kindly consider decreasing the spacing between the two division to have an exact dedign as the one given check the image below 👇 
<img width="1004" alt="Screen Shot 2023-03-31 at 15 12 15" src="https://user-images.githubusercontent.com/60146030/229130112-5b551a7f-a7b6-4981-8d6f-6480d61b616b.png">